src/Idris/IBC.hs

```diff
@@ -240,65 +289,68 @@ ibc i (IBCUsage n) f = return f { ibc_usage = n : ibc_usage f }
 ibc i (IBCExport n) f = return f { ibc_exports = n : ibc_exports f }
 ibc i (IBCAutoHint n h) f = return f { ibc_autohints = (n, h) : ibc_autohints f }
 
+snarf :: (Binary b, NFData b) => FilePath -> Archive -> Idris b
```

Comment:
This function name doesn't say much to me... is there a reason why it's not called something like `extractIBCPart`?